openapi_server/apis/users_api.py

```diff
@@ -50,8 +50,10 @@ async def create_user(
     try:
         DbUser.get(create_user.username)
         return Response(status_code=409)
-    except Exception as e:
+    except DbUser.DoesNotExist:
```

Comment:
we would still want to catch general exception with 500 error.

Reply:
Wouldn't the general exceptions be caught in line 55?

Reply:
oh I did not see that!